scripts/gen_mmu.py

```diff
@@ -101,6 +101,9 @@ def read_mmu_list_marshal_param():
                                                size_read_from_binary);
         size_read_from_binary += struct.calcsize(struct_mmu_regions_format);
 
+        if basic_mem_region_values[1] == 0:
```

Comment:
Looks fine, but since you're in this part of the code: `start_location` and `end_location` could be moved out of the inner loop below and this check could be `if start_location == end_location` instead.